### PR TITLE
Add chinese abbreviations for area,length,mass units

### PR DESCRIPTION
**Pull request by chinasqzl**
Add chinese abbreviations for area,length,mass units

### CODE REVIEW

File: Common/UnitDefinitions/Length.json [hunk 1]
```diff
@@ -23,6 +23,10 @@
         {
           "Culture": "ru-RU",
           "Abbreviations": [ "м" ]
+        },
+        {
+          "Culture": "zh-CN",
+          "Abbreviations": [ "米" ]
```
Comment (angularsen) [inline Common/UnitDefinitions/Length.json:29]: Can you please verify that the prefixes work as expected?
For example, 

- Does 毫米 read correctly as millimeter? 
- Does 千克 read correctly as kilogram?
- Does 兆吨 read correctly as megatonne?

Same for all the other units where "Prefixes" are used.
Google seems to agree, but it's best you verify.

If there are any issues with some units, where prepending the prefix does not read correctly, you can customize the abbreviation for a particular unit and prefix with "AbbreviationsForPrefixes", see examples here:

https://github.com/angularsen/UnitsNet/search?l=JSON&q=AbbreviationsForPrefixes

**Review by angularsen — APPROVED** [review]
This looks good to me, I just need you to verify the prefixes work as intended.

Update:
You need to run `generate-code.bat` to generate all the new unit abbreviations.

File: Common/UnitDefinitions/Length.json (hunk 1 above)
Comment (angularsen) [reply Common/UnitDefinitions/Length.json:29]: You need to run generate-code.bat to generate all the new unit abbreviations.

Comment (chinasqzl) [reply Common/UnitDefinitions/Length.json:29]: image

Comment (angularsen) [reply Common/UnitDefinitions/Length.json:29]: I take it you have reviewed these abbreviations then 👍

**Comment by angularsen** [comment]
Nuget is on the way out, thank you for the contribution!

[Release UnitsNet/4.79.0 · angularsen/UnitsNet](https://github.com/angularsen/UnitsNet/releases/tag/UnitsNet%2F4.79.0)

**Comment by angularsen** [comment]
@chinasqzl Hi, there is some recent discussion on reverting `厘米` back to `cm` and instead localizing `厘米` for the unit enum name "Centimeter".

Assuming you know Chinese, could you please help determine what is correct?

#1171 Localization related Issues

**Pull request merged**